CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Unreleased
 
 - Added `Window::focus_window`to bring the window to the front and set input focus.
+- On Web, added support for `Event::Touch`.
```

Comment:
```suggestion
- On Web, added support for `WindowEvent::Touch`.
- Added `Window::focus_window`to bring the window to the front and set input focus.
```

Reply:
You still need to make it say `WindowEvent::Touch` rather than `Event::Touch`

Reply:
Sorry, read over that change. Fixed it now!